src/scss/includes/search_form.scss

```diff
@@ -135,6 +140,19 @@ input[type="search"]::-webkit-search-cancel-button {
   color: $secondary_text;
   background-color: $background;
   transition: background-color .2s;
+  cursor: pointer;
+}
+
+.autocomplete_suggestion__first_line {
+  display: inline-block;
```

Comment:
On Firefxo, ellipsis does not work properly on this first line: the whole line will be collapsed to "..." because of this "inline-block".

Reply:
yep, fixed